_examples/8-questions-to-fit-candidate-into-team.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 8 Questions, to fit candidate into team
+author: Ievgenii Vdovenko
+role:  interviewer
+overview: Aim of this questions is to build happy team. My believe is - if team is happy, it's more productive.
+questions:
+- question: Describe your vision of role you applied to.
+  summary: very often candidate and company think differently about Role, so need sync up such detail; 
+- question: Deal breakers, or what situation/environment will immediately trigger red flag in your head.
+  summary: important information, because may be your env./team/etc. is bad for candidate, and could lead to future problems for you and candidate
```

Comment:
i would say: `important information, because maybe env./team/etc. is wrong for the candidate, and could lead to future problems for you and candidate`